ui/app/styles/helper-classes/spacing.scss

```diff
@@ -98,6 +99,10 @@
   margin: $spacing-4 0;
 }
 
+.has-top-bottom-margin-12 {
```

Comment:
since this is a new class, I opted to use the updated `-12` number naming which is a pattern we had discussed moving towards (and away from `s`, `m`, `l`) happy to revert and make this `-s` if folks prefer

Reply:
Can you just add a comment here to that affect? That this is the pattern we want to move toward, away from s/m/l so that future code-browsers will not be confused by the different types of measurements

Reply:
Yes, good idea! Will add